site_scons/components/__init__.py

```diff
@@ -221,50 +219,34 @@ def define_common(reqs):
     """common system component definitions"""
     reqs.define('cmocka', libs=['cmocka'], package='libcmocka-devel')
 
-    reqs.define('libunwind', libs=['unwind'], headers=['libunwind.h'],
-                package='libunwind-devel')
+    reqs.define('libunwind', libs=['unwind'], headers=['libunwind.h'], package='libunwind-devel')
 
     reqs.define('lz4', headers=['lz4.h'], package='lz4-devel')
 
-    reqs.define('valgrind_devel', headers=['valgrind/valgrind.h'],
-                package='valgrind-devel')
+    reqs.define('valgrind_devel', headers=['valgrind/valgrind.h'], package='valgrind-devel')
 
-    reqs.define('cunit', libs=['cunit'], headers=['CUnit/Basic.h'],
-                package='CUnit-devel')
+    reqs.define('cunit', libs=['cunit'], headers=['CUnit/Basic.h'], package='CUnit-devel')
 
-    reqs.define('python34_devel', headers=['python3.4m/Python.h'],
-                package='python34-devel')
-
-    reqs.define('libelf', headers=['libelf.h'], package='elfutils-libelf-devel')
-
-    reqs.define('tbbmalloc', libs=['tbbmalloc_proxy'], package='tbb-devel')
-
-    reqs.define('jemalloc', libs=['jemalloc'], package='jemalloc-devel')
-
-    reqs.define('boost', headers=['boost/preprocessor.hpp'],
-                package='boost-python36-devel')
+    reqs.define('boost', headers=['boost/preprocessor.hpp'], package='boost-python36-devel')
 
     reqs.define('yaml', headers=['yaml.h'], package='libyaml-devel')
 
     reqs.define('event', libs=['event'], package='libevent-devel')
 
-    reqs.define('crypto', libs=['crypto'], headers=['openssl/md5.h'],
-                package='openssl-devel')
+    reqs.define('crypto', libs=['crypto'], headers=['openssl/md5.h'], package='openssl-devel')
 
-    reqs.define('json-c', libs=['json-c'], headers=['json-c/json.h'],
-                package='json-c-devel')
+    reqs.define('json-c', libs=['json-c'], headers=['json-c/json.h'], package='json-c-devel')
 
-    if reqs.get_env('PLATFORM') == 'darwin':
-        reqs.define('uuid', headers=['uuid/uuid.h'])
-    else:
-        reqs.define('uuid', libs=['uuid'], headers=['uuid/uuid.h'],
-                    package='libuuid-devel')
+    reqs.define('uuid', libs=['uuid'], headers=['uuid/uuid.h'], package='libuuid-devel')
+
+    reqs.define('hwloc', libs=['hwloc'], headers=['hwloc.h'], package='hwloc-devel')
+
+    reqs.define('protobufc', libs=['protobuf-c'], headers=['protobuf-c/protobuf-c.h'],
+                package='protobuf-c-devel')
```

Comment:
This is the definitions for hwloc and protobufc.

I did remove a couple of definitions from this file but they weren't ones that we ever used or checked for.